Jenkinsfile

```diff
@@ -253,6 +258,46 @@ pipeline {
             }
         }
 
+        stage('QA') {
+            when {
+                expression { params.RUN_QA }
+            }
+            environment {
+                IMAGE = "gcr.io/zeebe-io/zeebe"
+                VERSION = readMavenPom(file: 'parent/pom.xml').getVersion()
+                TAG = "${env.GIT_COMMIT}"
+                DOCKER_GCR = credentials("zeebe-gcr-serviceaccount-json")
+                ZEEBE_AUTHORIZATION_SERVER_URL = 'https://login.cloud.ultrawombat.com/oauth/token'
+                ZEEBE_CLIENT_ID = 'W5a4JUc3I1NIetNnodo3YTvdsRIFb12w'
+                QA_RUN_VARIABLES = "{\"zeebeImage\": \"${IMAGE}:${TAG}\", \"generationTemplate\": \"${params.GENERATION_TEMPLATE}\", \"channel\": \"Internal Dev\"}"
+            }
+
+            steps {
+                container('docker') {
+                    sh 'cp dist/target/zeebe-distribution-*.tar.gz zeebe-distribution.tar.gz'
+                    sh '.ci/scripts/docker/build.sh'
+                    sh '.ci/scripts/docker/upload-gcr.sh'
+                    withVault(
+                      [ vaultSecrets:
+                        [
+                          [ path: 'secret/common/ci-zeebe/testbench-secrets-int',
+                            secretValues:
+                              [
+                                [envVar: 'ZEEBE_CLIENT_SECRET', vaultKey: 'clientSecret'],
+                                [envVar: 'ZEEBE_ADDRESS', vaultKey: 'contactPoint'],
+                              ]
+                          ],
+                        ]
+                      ]
+                    ) {
+                      sh '.ci/scripts/distribution/qa-testbench.sh'
+                    }
+                }
+
+                input message: 'Were all QA tests successful?', ok: 'Yes'
```

Comment:
this pauses the build. It asks the user for input

Reply:
I talked to Infra whether we can have some way of callback from the testbench and Infra declined. The problem is in going through Okta authentication. So this was the easiest way to pause the build and let the user resume it.

Reply:
What I hadn't realized, though, is that it doesn't play nice with the build timeout we have. It will only wait until the build timeout, which currently is ~45 minutes. So there is some work still to be done here.